migration_toolkit_for_containers/mtc-migrating-vms.adoc

```diff
@@ -0,0 +1,79 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="mtc-migrating-vms"]
+= Migrating virtual machine storage
+include::_attributes/common-attributes.adoc[]
+:context: mtc-migrating-vms
+
+toc::[]
+
+{mtc-full} ({mtc-short}) 1.8.4 supports storage migration for KubeVirt virtual machines (VMs).
+
+The migration can occur offline, when the VM is turned off, or online, when the VM is running.
+
+The main use case for this storage migration for KubeVirt VMs is migration from one storage class to another. For example, if you are rebalancing between different storage providers, or if there is new storage available that is better suited to the workload running inside the VM.
+
+== Supported persistent volume actions
+
+[width="100%",cols="31%,33%,36%",options="header",]
+|===
+|Action
+|Supported
+|Description
+
+|Copy
+|Yes
+a|
+. Create a new persistent volume (PV) in the same namespace.
+. Copy data from the source PV to the target PV, and modify the VM definition to point to the new PV.
+** If the `liveMigrate` flag is set, the VM migrates live.
+** If the `liveMigrate` flag is not set, the VM shuts down, and the source PV contents are copied to the target PV and the VM is restarted.
+
+|Move
+|No
+|This action is not supported.
+|===
+
+== Preparing for migration
+
+To prepare for migration, deploy mig-controller and Velero, and then create the following `Mig`' resources on the cluster where the migration controller is running to prepare for migration:
+
+.Procedure
+
+. Deploy link:https://github.com/kubevirt/kubevirt/[KubeVirt]. In order to support storage live migration, you need to deploy KubeVirt version 1.3.0 or later. Earlier versions do not support storage live migration. You also need to configure KubeVirt to enable storage live migration according to the link:https://kubevirt.io/user-guide/storage/volume_migration/[Volume Migration].
```

Comment:
I mention kubevirt 1.3.0 or later in the U/S documentation. This corresponds to OpenShift Virtualization 4.17 or newer. Might want to say that instead of KubeVirt 1.3.0. But since U/S doesn't know about OCP-V I said KubeVirt there.

I am not sure if there is a link to D/S documentation for storage live migration. For now the U/S version is fine, but once I know there is a D/S version we probably want to change it to the D/S link.